test/repository/MockCache.scala

```diff
@@ -50,6 +50,9 @@ trait MockCache extends MockitoSugar with BeforeAndAfterEach {
   protected def givenTheCacheIsEmpty(): Unit =
     given(cache.findByPid(any())).willReturn(Future.successful(None))
 
+  protected def givenTheCacheIsCleared(): Unit =
```

Comment:
Dont think we need a method for this, we can just stick it in the `beforeEach`

Reply:
Done